channeldb/invoice_test.go

```diff
@@ -439,9 +438,8 @@ func TestInvoiceCancelSingleHtlc(t *testing.T) {
 		},
 	}
 
-	if _, err := db.AddInvoice(testInvoice, paymentHash); err != nil {
-		t.Fatalf("unable to find invoice: %v", err)
-	}
+	err = db.AddInvoice(testInvoice)
+	require.NoError(t, err)
```

Comment:
We want to use what makes more sense here (`require.NoError`). All new checks are using the `NoError` method but I did not update all the old ones. It will be updated organically whenever we touch this files or in another PR that changes that for this and many other files.